alea/include/alps/alea/internal/joined.hpp

```diff
@@ -18,12 +18,49 @@ namespace alps { namespace alea {
 
     struct circular_var;
     struct elliptic_var;
+
 }}
 
+namespace alps { namespace alea { namespace internal {
+    template <typename... Pack> struct add_scalar;
+    template <typename... Pack> struct joined_value;
+}}}
+
 // Actual declarations
 
 namespace alps { namespace alea { namespace internal {
 
```

Comment:
Please put comments abut what this does

Reply:
These are only forward declarations, the documentation is with the actual declaration.